MonoGame.Framework.Content.Pipeline/Builder/PipelineBuildEvent.cs

```diff
@@ -176,7 +185,7 @@ public bool NeedsRebuild(PipelineManager manager, PipelineBuildEvent cachedEvent
 
             // If the source file is newer than the dest file
             // then it must have been updated and needs a rebuild.
-            if (sourceWriteTime >= destWriteTime)
+            if (sourceWriteTime > destWriteTime)
```

Comment:
The comment was already there. And it does accurately describe what the code was "suppose" to be doing. Technically this line is fixing a preexisting bug.